data/mods/Magiclysm/items/enchanted_melee.json

```diff
@@ -752,11 +752,11 @@
     "price": 5000,
     "bashing": 27,
     "cutting": 1,
-    "material": [ "steel", "glass" ],
+    "material": [ { "type": "steel", "portion": 199 }, { "type": "glass" } ],
```

Comment:
Wanting to confirm that portion size works over 100?    So this would be 99.5% steel with .5% glass?

Reply:
I took a look at https://github.com/CleverRaven/Cataclysm-DDA/pull/52369 and I didn't see any reason it wouldn't work with large values. I just loaded up the game with the change and it seemed to be working fine, I could smash stuff with the mana-toolbar and it wasn't shattering. Yeah, .5% glass, which I think is reasonable.

Reply:
Eh, it looks like it's still far too likely to break when used to attack at those proportions. That being the case and given that the intent appears to be for it to be a viable weapon as well I'm going to set the material to steel alone.